### ISSUE TITLE
HTTPError: 403 (Forbidden) in Travis

### ISSUE THREAD

**Issue by asamuzaK** [nexe/nexe#598]
**Is this a BUG or a FEATURE REQUEST?:** 
Bug

**What happened:**
Build failed due to 403.
```
- Starting...
ℹ nexe 3.0.0
- nexe 3.0.0
✔ Downloading pre-built Node.js
- Finished in 1.483s
✔ Finished in 1.483s
HTTPError: Response code 403 (Forbidden)
    at EventEmitter.emitter.on (/home/travis/build/asamuzaK/withExEditorHost/node_modules/got/source/as-promise.js:74:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! withexeditorhost@5.0.7 package:linux64: `nexe ./index.js -t linux-x64-10.15.3 -o ./pkg/linux64/index`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the withexeditorhost@5.0.7 package:linux64 script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-04-01T14_15_55_564Z-debug.log
The command "mkdir ./pkg/linux64 && npm run package:linux64 && chmod 755 ./pkg/linux64/index && npm run zippackage:linux64" failed and exited with 1 during .
Your build has been stopped.
```

Full log [Job #720.5 - asamuzaK/withExEditorHost - Travis CI](https://travis-ci.org/asamuzaK/withExEditorHost/jobs/514162410)

**What you expected to happen:**
Build success.

**How to reproduce it (as minimally and precisely as possible):**

**Anything else we need to know?:**

**Environment**
See linked log above.

- Platform(OS/Version):
- Host Node Version:
- Target Node Version:
- Nexe version:
- Python Version:

**Comment by calebboyd** [comment]
I'm going to leave this open to track getting rid of that github API request...should just need the releases which are not authenticated at all.

**Comment by calebboyd** [comment]
Fixed in 3.0.1

**Comment by asamuzaK** [comment]
Thanks!
